src/api/Annotations.js

```diff
@@ -164,16 +187,85 @@ export default class Annotations extends MarkerBasedApi {
             return;
         }
 
+        const requestData = {
+            file_id: fileId,
+            file_version_id: fileVersionId,
+            ...(shouldFetchReplies ? { fields: 'replies' } : null),
```

Comment:
minor: I think this could be simplified a little:
```suggestion
            ...(shouldFetchReplies && { fields: 'replies' }),
```

Reply:
I've tried it and I am afraid flow does not like it: 
`boolean [1] is not an object.`